### PR TITLE
feat: gate bazed-tauri behind a feature-flag

### PR DESCRIPTION
**Pull request by elkowar**

### CODE REVIEW

File: crates/bazed/src/main.rs [hunk 1]
```diff
@@ -34,7 +34,10 @@ async fn main() -> Result<()> {
     });
 
     if run_frontend {
+        #[cfg(feature = "tauri")]
         bazed_tauri::run_frontend();
+        #[cfg(not(feature = "tauri"))]
+        eprintln!("please compile with the `tauri`-feature enabled or pass --no-frontend");
```
Comment (buffet) [inline crates/bazed/src/main.rs:40]: panic! would be better, no?

**Review by buffet — CHANGES_REQUESTED** [review]

File: crates/bazed/src/main.rs (hunk 1 above)
Comment (elkowar) [reply crates/bazed/src/main.rs:40]: would it be? I don't really see why we should panic here if we don't have to :/

Comment (buffet) [reply crates/bazed/src/main.rs:40]: We also can't do what the user wants in any way.

Comment (elkowar) [reply crates/bazed/src/main.rs]: I guess? Still not a fan, but lemme just panic

**Review by buffet — APPROVED** [review]

**Pull request merged**